apps/babanews/src/styles/theme.ts

```diff
@@ -0,0 +1,47 @@
+import {createTheme} from '@mui/material'
+
+const {
+  palette: {augmentColor}
+} = createTheme()
+
+const theme = createTheme({
```

Comment:
We should inherit from the WePtheme

```suggestion
import {theme as WePTheme} from "@wepublish/ui"

const theme = createTheme(WePTheme, {
```